### DISCUSSION TITLE
How to create create a normal send transaction on bitcoin wallet using bitcoinjs-lib

### DISCUSSION THREAD

**Discussion by Surya-Kant-Sharma** [phantom/docs#274] [General]
I am currently beginning my journey in Bitcoin development and am trying to familiarize myself with bitcoinjs-lib. However, I have encountered some errors in my initial attempts to implement certain functionality using this library.

Could you kindly provide me with the correct code or guidance to help resolve these issues? Your assistance would be greatly appreciated as I continue to develop my skills in this area.

**Comment by adamdelphantom** [comment]
Hello @Surya-Kant-Sharma,

We do not provide generalized how-to support for development. Please see [our documentation](https://docs.phantom.app/bitcoin/integrating-phantom) for developing with Phantom and Bitcoin.
